$OpenBSD: patch-acsite_m4,v 1.1 2017/09/28 19:16:36 dcoppa Exp $

Get the list of mounted filesystems: this isn't used outside of
configure, so just skip the broken tests.

Index: acsite.m4
--- acsite.m4.orig
+++ acsite.m4
@@ -9,7 +9,7 @@ getcwd getmntinfo gettimeofday isascii lchown \
 listmntent memcpy mkfifo strchr strerror strrchr)
 
 # Determine how to get the list of mounted filesystems.
-list_mounted_fs=
+list_mounted_fs=no
 
 # If the getmntent function is available but not in the standard library,
 # make sure LIBS contains -lsun (on Irix4) or -lseq (on PTX).
